### ISSUE TITLE
-m is broken on srt files

### ISSUE THREAD

**Issue by OilSubjectLoss7** [helixarch/subedit#19]
Example:
Og:

```
00:12:24,541 --> 00:12:27,166

- We're up. Are you sure about this?

- Of course not.
```

subed -i file -m applied:
```
00:12:24,541 --> 00:12:27,166
- We're up. Are you sur
- Of course not.
e about this?
```

or:

Og

```
00:12:24,541 --> 00:12:27,166

- We're up. Are you gay?

- Of course.
```

subedit -i file -m applied:
```
00:12:24,541 --> 00:12:27,166

- We're up. Are

- Of course.
```

So, two bugs.
1st. Cut the first line and put it in the end.
Making this mess:
```
00:12:24,541 --> 00:12:27,166
- We're up. Are you sur
- Of course not.
e about this?
```
The second bug just delete half of the line completely:
```
00:12:24,541 --> 00:12:27,166

- We're up. Are

- Of course.
```
This doesn't happened when i do it with sub format subtitle.
So i had to apply the `-m` on sub file first then convert to them back srt.
But sub format subtitle are rare so i have to convert my srt files to sub first then, apply `-m` then convert back to srt.

**Comment by OilSubjectLoss7** [comment]
Nevermind, it's on my end ```fd -e srt -x subedit -i {} {-c, -m}``` seems to be at fault here, but would you mind fixing it either way, cause ```fd -e sub -x subedit -i {} {-c, -m}``` works as intended


𝗲𝗱𝗶𝘁:
scratch that. even without using `fd` i sitll get the buggy mess

**Comment by OilSubjectLoss7** [comment]
Found it, it has to be run in an empty dir except the subtitle itself. The program seems to print the directory files inside the srt file. Weird bug, please fix

edit:
Nope it even prints out the subtitle filename inside the subtitle. Please fix

**Comment by ody1** [comment]
If you need to run subedit at all the srt files in a directory, use it like this:
$ subedit -i "*.srt" -c -m -b
For multiple examples for all use cases see:
$ subedit -H

If you use subedit like above and still experiencing bugs, or if you are trying to do something else, please elaborate and send an example subtitle, before and after you use subedit on it.

**Comment by OilSubjectLoss7** [comment]
False alarm, my machine was at fault, thanks for the tool mate

**Comment by OilSubjectLoss7** [comment]
This still happened with clean slate machine and on other machine. Sub file works really well but srt don't.
Weird bug as we know this is really made for srt file as priority.

[sample.zip](https://github.com/helixarch/subedit/files/8961502/sample.zip)

> Example: Og:
> 
> ```
> 00:12:24,541 --> 00:12:27,166
> 
> - We're up. Are you sure about this?
> 
> - Of course not.
> ```
> 
> subed -i file -m applied:
> 
> ```
> 00:12:24,541 --> 00:12:27,166
> - We're up. Are you sur
> - Of course not.
> e about this?
> ```
> 
> or:
> 
> Og
> 
> ```
> 00:12:24,541 --> 00:12:27,166
> 
> - We're up. Are you gay?
> 
> - Of course.
> ```
> 
> subedit -i file -m applied:
> 
> ```
> 00:12:24,541 --> 00:12:27,166
> 
> - We're up. Are
> 
> - Of course.
> ```
> 
> So, two bugs. 1st. Cut the first line and put it in the end. Making this mess:
> 
> ```
> 00:12:24,541 --> 00:12:27,166
> - We're up. Are you sur
> - Of course not.
> e about this?
> ```
> 
> The second bug just delete half of the line completely:
> 
> ```
> 00:12:24,541 --> 00:12:27,166
> 
> - We're up. Are
> 
> - Of course.
> ```
> 
> This doesn't happened when i do it with sub format subtitle. So i had to apply the `-m` on sub file first then convert to them back srt. But sub format subtitle are rare so i have to convert my srt files to sub first then, apply `-m` then convert back to srt.

**Comment by OilSubjectLoss7** [comment]
my bad, as it turns out my bash was outdated,

**Comment by OilSubjectLoss7** [comment]
nope, still happens. i still need the convert to sub format intermediate stage.
